pkgs/servers/nosql/keydb/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libuuid, curl
+, openssl
+, pkg-config
+, systemd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keydb";
+  version = "6.3.1";
+
+  src = fetchFromGitHub {
+    owner = "snapchat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-00Dx6GRAXVpJUXhu4kjsVCEsoN615vhGvc7+gViERqI=";
+  };
+
```

Fix cross-compilation:
```suggestion
  postPatch = ''
    substituteInPlace deps/lua/src/Makefile \
      --replace "ar rcu" "${stdenv.cc.targetPrefix}ar rcu"
    substituteInPlace src/Makefile \
      --replace "as --64 -g" "${stdenv.cc.targetPrefix}as --64 -g"
    substituteInPlace deps/Makefile \
      --replace "cd jemalloc && ./configure" "cd jemalloc && ./configure --host=${stdenv.buildPlatform.config}"
  '';

```
It would be better to make a proper PR to upstream and use `fetchpatch` here.